website/src/index.html

```diff
@@ -1,26 +1,59 @@
-<!doctype html>
+<!DOCTYPE html>
 <html>
-	<head>
-		<title>Apiary Explorer | apiary.1hive.org </title>
-		<!-- Accessibility -->
-		<meta name="viewport" content="width=device-width, initial-scale=1">
-		<meta charset="UTF-8" />
-		<!-- SEO -->
-		<meta name="description" content="Apiary Explorer allows you to explore and search Aragon DAOs on the Ethereum network">
-		<!-- Open Graph -->
-		<meta property="og:title" content="Apiary Explorer" />
-		<meta property="og:description" content="Apiary Explorer allows you to explore and search Aragon DAOs on the Ethereum network" />
-		<meta property="og:url" content="https://apiary.1hive.org" />
-		<!-- Twitter -->
-		<meta name="twitter:creator" content="@1hiveorg" />
-		<!-- Misc !-->
-	</head>
+  <head>
+    <title>Apiary Explorer | apiary.1hive.org</title>
+    <!-- Accessibility -->
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta charset="UTF-8" />
+    <!-- SEO -->
+    <meta
+      name="description"
+      content="Apiary Explorer allows you to explore and search Aragon DAOs on the Ethereum network"
+    />
+    <!-- Open Graph -->
+    <meta property="og:title" content="Apiary Explorer" />
+    <meta
+      property="og:description"
+      content="Apiary Explorer allows you to explore and search Aragon DAOs on the Ethereum network"
+    />
+    <meta property="og:url" content="https://apiary.1hive.org" />
+    <!-- Twitter -->
+    <meta name="twitter:creator" content="@1hiveorg" />
+    <!-- Misc !-->
+  </head>
 
-	<body>
-		<noscript>Your browser does not support JavaScript. Please update to a modern browser to use Apiary.</noscript>
+  <body>
+    <noscript
+      >Your browser does not support JavaScript. Please update to a modern
```

Comment:
Typo

Reply:
Done in https://github.com/1Hive/apiary/pull/188/commits/df0d82f150aef65bc514fbdf9d65d765a0051a76